File: tests/fixtures/multilang/sample.ts
import { helper } from './helper';

/**
 * Adds two numbers.
 */
function add(a: number, b: number): number {
  if (a > 0) {
    return helper.scale(a, b);
  }
  return a + b;
}

class Widget {
  // Renders the widget.
  render(): void {
    this.add(1, 2);
    add(3, 4);
  }
}

File: tests/fixtures/multilang/sample.rb
require "helper"

# Adds two numbers.
def add(a, b)
  if a > 0
    helper.scale(a, b)
  else
    a + b
  end
end

class Widget
  # Renders the widget.
  def render
    add(1, 2)
  end
end
